pkg/skaffold/deploy/helm.go

```diff
@@ -106,8 +107,12 @@ func (h *HelmDeployer) deployRelease(out io.Writer, r v1alpha2.HelmRelease, b *b
 		args = append(args, "upgrade", r.Name, r.ChartPath)
 	}
 
-	if r.Namespace != "" {
-		args = append(args, "--namespace", r.Namespace)
+	ns := r.Namespace
```

Comment:
Is there a way to map the cli arguments to env variables? I think there is. It would be much better to handle that once for all when CLI arguments are parsed.

Reply:
good point yeah; in this case we're merging default values from the CLI / env var overriding any missing values in the `skaffold.yaml`